examples/python/Vision_files_preprocess.py

```diff
@@ -0,0 +1,623 @@
+
+    fig, ax = plt.subplots(figsize = (8,6))
+
+    ax.plot(np.mean(prompts_precorr_arr[TOF_bin, central_slice-thickness_half:central_slice+thickness_half, 0, :], axis=(0)), label='Prompts, pre-corrected f. multi. factors')
+    ax.plot(np.mean(additive_term_arr[TOF_bin, central_slice-thickness_half:central_slice+thickness_half, 0, :], axis=(0)), label='additive term')
+
+    ax.set_xlabel('Radial distance (bin)')
+    ax.set_ylabel('total counts')
+    ax.set_title('TOF bin:' + str(TOF_bin))
+    ax.legend()
+    plt.tight_layout()
+    plt.suptitle('Lineprofiles - Avg over 10 central slices')
+    plt.savefig(os.path.join(STIR_output_folder,'additive_term_lineprofiles.png'), transparent=False, facecolor='w')
+    plt.tight_layout()
+
+    # %%
+    #### PLOT BACKGROUND TERM
+    #### draw line-profiles to check if all's correct
+    prompts_arr = stirextra.to_numpy(prompts_from_e7)
+    BG_arr = scatter_3D_unnormalized_arr + randoms_arr
+
+    #%%
+    fig, ax = plt.subplots(figsize = (8,6))
+
+    ax.plot(np.mean(prompts_arr[TOF_bin, central_slice-thickness_half:central_slice+thickness_half, 0, :], axis=(0)), label='Prompts')
+    ax.plot(np.mean(BG_arr[TOF_bin, central_slice-thickness_half:central_slice+thickness_half, 0, :], axis=(0)), label='BG term')
+
+    ax.set_xlabel('Radial distance (bin)')
+    ax.set_ylabel('total counts')
+    ax.set_title('TOF bin:' + str(TOF_bin))
+    ax.legend()
+    plt.tight_layout()
+    plt.suptitle('Lineprofiles - Avg over 10 central slices')
+    plt.savefig(os.path.join(STIR_output_folder,'BG_term_lineprofiles.png'), transparent=False, facecolor='w')
+    plt.tight_layout()
+    # %%
+
+
+
+def change_datafilename_in_interfile_header(new_header_filename, header_filename, data_filename):
+    with open(header_filename) as f:
+        data = f.read()
+    poss = re.search(r'name of data file\s*:=[^\n]*', data).span()
+    data = data.replace(data[poss[0]:poss[1]], \
+        'name of data file:={}'.format(data_filename))
+    with open(new_header_filename, 'w') as f2:
+        f2.write(data)
+
+def DOI_adaption(projdata, DOI_new):
+    proj_info = projdata.get_proj_data_info()
+
+    DOI = proj_info.get_scanner().get_average_depth_of_interaction()
+    print('Current Depth of interaction:', DOI)
+    proj_info.get_scanner().set_average_depth_of_interaction(DOI_new)
+    DOI = proj_info.get_scanner().get_average_depth_of_interaction()
+    print('New Depth of interaction:', DOI)
+
+def check_if_compressed(header_filename):
+    with open(header_filename) as f:
+        data = f.read()
+    try:
+        match = re.search(r'compression\s*:=\s*(\w+)', data)
+        if match.group(1) == 'off':
+            print('Compression is off, can proceed')
+        else:
+            print('You are trying to read e7tools compressed data. Please uncompress first!')
+            sys.exit()
+    except:
+        print('No compression info found in header!')
+        pass
```

Comment:
```suggestion
    else:
        print('No compression info found in header!')
```
Though maybe should be a warning